src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -370,6 +383,60 @@ module Types = struct
             ~resolve:(fun _ {Transaction_snark_work.Info.work_ids; _} ->
               One_or_two.to_list work_ids ) ] )
 
+  let signed_fee =
+    obj "SignedFee" ~doc:"Signed fee" ~fields:(fun _ ->
+        [ field "sign" ~typ:(non_null string) ~doc:"+/-"
```

Comment:
Noticed we use an enum for this in the ocaml code, seems like it would be appropriate to use a graphql enum for it as well, unless there was a reason you didn't want to.